ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -118,6 +122,21 @@ Rectangle {
         }
     }
 
+    Connections {
+        enabled: control.emojiPopupOpened
+        target: emojiPopup || null
```

Comment:
simple assigning emojiPopup would work?

Reply:
if its same reason here we can do a null check? I feel this more difficult tor read